lumberdev/atelier
# Feat/Drop Campaign from URL

### WHAT IT DOES:

- Removes `campaign` from URL
  -  Before: http://river.localhost:3000/campaign/our-shopify-store
  -  Now: http://river.localhost:3000/our-shopify-store
- Prevents merchant from creating store with handle `app`,`api` or `exitframe` as these would clash with our other endpoints
- Converts user handle input to lowercase and converts any spaces to dashes so handle input is valid for URLs 

### WHAT TO TEST:

- Check you can create a campaign successfully
- Check you are not allowed to create a campaign with the a handle of  `app`,`api` or `exitframe` 
- Check you cannot create any other invalid handle
- **Check there is no other instance in the codebase where it refers to `/campaign`**
